Gemfile

```diff
@@ -20,12 +20,15 @@ gem 'oai'
 gem 'ougai', '~> 1.8'
 gem 'parse-cron'
 gem 'puma', '~> 3.12'
-gem 'rails', '~> 5.2.3'
+gem 'rails', '~> 5.2.4'
 gem 'responders'
 gem 'sidekiq', '~> 5.2.3'
 gem 'sinatra', require: nil
-gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', tag: 'v2.8.0'
+gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', branch: 'rm/fix-secutity-scans-on-worker'
```

Comment:
When does this get merged and tagged?

Reply:
I'll merge and tag sj common when the story is accepted :)